src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -555,6 +556,28 @@ class AemConfig(
             "*_x0040_TypeHint"
     )
 
+    /**
+     * Define here mixin types that will be skipped when pulling JCR content from AEM instance.
+     */
+    @Input
+    var cleanSkipMixinTypes: MutableList<String> = mutableListOf(
+            "cq:ReplicationStatus",
+            "mix:versionable"
+    )
+
+    /**
+     * Define hook method for normalizing lines.
+     */
+    @Internal
+    @get:JsonIgnore
+    var cleanLineProcess: (VltCleaner, File, String) -> String = { cleaner, file, line -> cleaner.normalizeLine(file, line) }
```

Comment:
it is possible also to declare it as

```
 var cleanLineProcess: (VltCleaner) -> (File, String) -> String = { { file, line -> it.normalizeLine(file, line) } }
```

but looks almost same, so I am convinced that we could go with your propopsal. changing `this` is possible but I am afraid that we will be forced to declare it as `Closure<*>` so that it will be incompatible with Kotlin DSL / I am trying to make it even now compatible with Kotlin DSL which will be default in Gradle future.